src/explore-education-statistics-admin/src/pages/prototypes/components/PrototypeEditableContent.module.scss

```diff
@@ -0,0 +1,48 @@
+.editable-content {
+  cursor: pointer;
+  margin: 0;
+  padding: 0;
+  position: relative;
+
+  .editable-button {
+    background-color: rgba(240, 240, 240, 0.75);
+    display: block;
+    height: 100%;
+    padding: 0 4px;
+    position: absolute;
+    width: 100%;
+  }
+
+  .editable-button::after {
+    content: 'Click to edit this section';
```

Comment:
Text content like this in CSS doesn't seem like a good idea, can this be refactored into the markup?

Reply:
Moved into the code and added a nested div to handle the positioning.